sympy/functions/elementary/miscellaneous.py

```diff
@@ -340,7 +340,7 @@ def real_root(arg, n=None, evaluate=None):
 class MinMaxBase(Expr, LatticeOp):
     def __new__(cls, *args, **assumptions):
         if not args:
-            raise ValueError("The Max/Min functions must have arguments.")
+            cls._eval_no_args()
```

Comment:
This is a "do nothing" change since you don't return the result. So apparently the existing code works when there are no args as long as you don't raise an error. So you could just delete those two lines or accept a shortcut and `return cls._eval_no_args()`.

Reply:
Ohh I am sorry I commited it in my local repo and pushed it but i think it was not updated here due to low network.

Reply:
These lines can be totally removed because the case of no arguments is handled later:

        if not args:
            return cls.identity

Reply:
Ohh okk i did not see that. I thought it is handled at the time of creation.

Reply:
Is it Okk now?

Reply:
Good catch. I didn't realize we already had the identity set properly.

Reply:
Me too did'nt see down there